extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -131,12 +125,28 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
     }
   }
 
-  private def regexp_replace(expr: String, hasLen: Boolean = false): String = {
+  // public for test
```

Comment:
not necessary, you can add tests using sql queries

Reply:
Thank you for your feedback. I have made the changes you suggested. Please let me know if there is anything else I can do to improve this PR.